lib/ansible/cli/__init__.py

```diff
@@ -453,6 +453,10 @@ def _play_prereqs():
             loader.set_basedir(basedir)
             add_all_plugin_dirs(basedir)
             set_collection_playbook_paths(basedir)
+            default_collection = get_collection_name_from_path(basedir)
+            if default_collection:
+                display.warning("running adhoc inside collection {0}".format(default_collection))
```

Comment:
Is there a reason this is a warning instead of info or debug?  Also, you probably need to use `u""` literal strings since you're using `.format()`

```suggestion
                display.warning(u"running adhoc inside collection {0}".format(default_collection))
```

Reply:
Just consistency with the same behavior on the playbook side of things. It's a "weird" enough thing that I didn't want to catch someone unawares that it was happening. We can always back that off later if people complain...